src/Google.Api.Gax.Grpc/ApiCall.cs

```diff
@@ -12,6 +12,15 @@
 
 namespace Google.Api.Gax.Grpc
 {
+    /// <summary>
+    /// Delegate representing exception customization. This allows specific RpcExceptions to be
+    /// customized, by returning a new RpcException to replace the original. If the original
+    /// exception should be passed through, the delegate should return <c>null</c>.
+    /// </summary>
+    /// <param name="original">The original exception thrown by gRPC.</param>
+    /// <returns>A replacement exception, or <c>null</c> if no customization should be performed.</returns>
+    public delegate RpcException ExceptionCustomizer(RpcException original);
```

Comment:
Is this now required? Why not use a Func instead?

Reply:
Ooh, of course. Yay.